src/VisualStudio/Core/Def/PackageRegistration.pkgdef

```diff
@@ -13,3 +13,8 @@
 "CS"="CA719A03-D55C-48F9-85DE-D934346E7F70"
 "VB"="EEC3DF0D-6D3F-4544-ABF9-8E26E6A90275"
 
+[$RootKey$\FeatureFlags\Roslyn\LSP\Completion]
+"Description"="Enables the LSP-powered C#/VB completion experience to operate with prototype behavior."
+"Value"=dword:00000000
+"Title"="Enable experimental C#/VB LSP completion experience"
```

Comment:
This sounds like it adds LSP completion experiences to normal C# scenarios which isn't true. Maybe it's worth being extra clear in the description?